ivy/src/main/java/sbt/mavenint/SbtPomExtraProperties.java

```diff
@@ -12,7 +12,11 @@
     public static final String POM_INFO_KEY_PREFIX = "info.";
     public static final String POM_SCALA_VERSION = "scalaVersion";
     public static final String POM_SBT_VERSION = "sbtVersion";
+    /** @deprecated Use {@link #POM_SCALA_API_KEY} instead */
+    @Deprecated
```

Comment:
FWIW, we _do_ intend to honor `info.apiURL` as fallback to `info.scalaApiURL` for pre-existing poms. So we are good on that front.

However, there could potentially be case where you publish an upstream lib with sbt 0.13.8 (and thus have scalaApiURL), however downstream library might have been using sbt 0.13.6 which won't see apiURL.

So, the point @eed3si9n is making is valid and we can stay with `info.apiURL` if we _must_ at the cost of adding to the overall non-intuitiveness :)